nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/MetricsDescriptor.java

```diff
@@ -140,16 +160,22 @@ public String prefix() {
         }
 
         protected boolean filter(String name) {
-            if (filters == null || filters.isEmpty()) {
-                String filters = Framework.getProperty(FILTER_PROPERTY, DEFAULT_METRICS);
-                return matchesFilter(name, Arrays.stream(filters.split(",")));
+            if ((whitelistMetrics == null || whitelistMetrics.isEmpty())
```

Comment:
You check for nullity here for both elements, nothing ensure you that `whitelistMetrics == null || blacklistMetrics == null`.
To get rid of nullity check, we can instantiate our lists with `Collections.emptyList()`, code will be cleaner and more readable.